prow/crier/controller.go

```diff
@@ -181,7 +182,7 @@ func (c *Controller) updateReportState(pj *v1.ProwJob) error {
 		logrus.Warnf("Empty merge patch: pjData: %s, newpjData: %s", string(pjData), string(newpjData))
 	}
 
-	logrus.Infof("Created merge patch: %v", string(patch))
+	log.Infof("Created merge patch: %v", string(patch))
```

Comment:
Should this be `Info`? I sure don't want to see a merge patch .... I imagine we log what we were *trying* to do in English, which should be good enough.

Reply:
But why not, they are interesting?

JK, I removed it, it was cargoed from before.